### DISCUSSION TITLE
Email confirmation pages text

### DISCUSSION THREAD

**Discussion by AlexBab11** [keycloakify/keycloakify#715] [Q&A]
Hi, I can't find keys for changing descriptions in two pages, here is an example. Could you please give advice on this? I also don't know how to reproduce this behaviour locally, or using story book because both pages seem to be info.ftl. 
![image](https://github.com/user-attachments/assets/6cbe013d-99be-42dc-a8f3-2dc98aef9964)
![image](https://github.com/user-attachments/assets/9388ec45-13a5-45d8-99d4-682bba325525)

**Comment by garronej** [comment]
Hello @AlexBab11,  

Please take some time reading the documentation, you should be able to figure it out:  
https://docs.keycloakify.dev/i18n/base-principles  
https://docs.keycloakify.dev/i18n/adding-new-translation-messages-or-changing-the-default-ones  

If you still have issue ping me
